internal/v5/api.go

```diff
@@ -235,6 +235,7 @@ func RouterHandlers(h *ReqHandler) *router.Handlers {
 			"perm":             h.puttableBaseEntityHandler(h.metaPerm, h.putMetaPerm, "acls", "developmentacls"),
 			"perm/":            h.puttableBaseEntityHandler(h.metaPermWithKey, h.putMetaPermWithKey, "acls", "developmentacls"),
 			"promulgated":      h.baseEntityHandler(h.metaPromulgated, "promulgated"),
+			"resources":        h.EntityHandler(h.metaResources, "_id", "charmmeta"),
```

Comment:
We use Entity.URL (which is _id in the doc).  Do you mean that's always filled in?  I noticed that most of the other handlers require _id.